### DISCUSSION TITLE
Ethos

### DISCUSSION THREAD

**Discussion by quickcarl42** [FrSkyRC/ETHOS-Feedback-Community#2899] [Q&A]
hi just got a Rxsky x20s radio yesterday and trying to setup Ethos suite. suite does not do anythig says no radio connected.
what do i have to do any help would be great

**Comment by mipel85** [comment]
Hello,
What operating system is used?
Under Linux Mint, I use a virtual machine to run Ethos Suite and my radio is not always recognized the first time.
You must connect it in bootloader mode first before launching the software.
Hope this can help.
